### PR TITLE
Allow Vagrant user to access app_dev.php

### PR DESCRIPTION
**Pull request by Netsensia**

### CODE REVIEW

File: web/app_dev.php [hunk 1]
```diff
@@ -7,14 +7,16 @@
...
 
-// This check prevents access to debug front controllers that are deployed by accident to production servers.
-// Feel free to remove this, extend it, or make something more sophisticated.
-if (isset($_SERVER['HTTP_CLIENT_IP'])
-    || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
-    || !(in_array(@$_SERVER['REMOTE_ADDR'], array('127.0.0.1', 'fe80::1', '::1')) || php_sapi_name() === 'cli-server')
-) {
-    header('HTTP/1.0 403 Forbidden');
-    exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
+if (strtolower($_SERVER['HTTP_HOST']) != 'pipelinedashboard.local') {
```
Comment (brettminnie) [inline web/app_dev.php]: you shouldn't have to modify this, have you tried setting the var in the .htaccess file?

Comment (Netsensia) [inline web/app_dev.php]: Which var?

Comment (brettminnie) [inline web/app_dev.php]: Have you had a poke here
http://symfony.com/doc/current/cookbook/configuration/web_server_configuration.html

File: src/DashboardHub/Bundle/AppBundle/Controller/DefaultController.php [hunk 2]
```diff
@@ -12,6 +12,7 @@ class DefaultController extends Controller
 {
     public function indexAction()
     {
+        echo $this->container->getParameter('kernel.environment'); die;
```
Comment (eddiejaoude) [inline src/DashboardHub/Bundle/AppBundle/Controller/DefaultController.php]: Debug code?

File: src/DashboardHub/Bundle/AppBundle/Menu/MenuBuilder.php [hunk 3]
```diff
@@ -46,6 +46,7 @@ public function createMainMenu(Request $request)
             $menu->addChild('Logout ' . $this->securityContext->getToken()->getUser()->getUsername(), array('route' => 'logout'));
         } else {
             $menu->addChild('Login', array('route' => 'login'));
+            //$menu->addChild(, array('route' => 'login'));
```
Comment (eddiejaoude) [inline src/DashboardHub/Bundle/AppBundle/Menu/MenuBuilder.php]: ?

**Pull request closed without merging**